pyvdrm/drm.py

```diff
@@ -77,6 +77,14 @@ def __repr__(self):
         return "{} {} {}".format(arg1, type(self), arg2)
 
 
+class AsiMultipleExpr(AsiExpr):
+    """Subclass with syntactic sugar for boolean ops"""
```

Comment:
Should the docstring get updated?

Reply:
No, but I deleted the other (unused) class with the same docstring instead.